### PR TITLE
Shade and minimize jar and remove Guava

### PR DESCRIPTION
**Pull request by dbmeneses**
Reduces artifact size from 3.8MB to 1.6MB.

### CODE REVIEW

File: sonar-python-plugin/src/main/java/org/sonar/plugins/python/PythonProfile.java [hunk 1]
```diff
@@ -56,12 +57,13 @@ private void loadActiveKeysFromJsonProfile(RulesProfile rulesProfile) {
   }
 
   public static Set<String> activatedRuleKeys() {
-    URL profileUrl = PythonProfile.class.getResource("/org/sonar/l10n/py/rules/python/Sonar_way_profile.json");
-    try {
+    String path = "/org/sonar/l10n/py/rules/python/Sonar_way_profile.json";
```
Comment (SonarTech) [inline sonar-python-plugin/src/main/java/org/sonar/plugins/python/PythonProfile.java]: MINOR Refactor your code to get this URI from a customizable parameter. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1075)

File: python-checks/src/main/java/org/sonar/python/checks/SameBranchCheck.java [hunk 2]
```diff
@@ -128,8 +128,8 @@ private static AstNode singleIfChild(AstNode suite) {
     List<AstNode> statements = suite.getChildren(PythonGrammar.STATEMENT);
     if (statements.size() == 1) {
       AstSelect nestedIf = statements.get(0).select()
```
Comment (SonarTech) [inline python-checks/src/main/java/org/sonar/python/checks/SameBranchCheck.java]: MINOR Remove this use of "AstSelect"; it is deprecated. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)
MINOR Remove this use of "select"; it is deprecated. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)

File: sonar-python-plugin/src/main/java/org/sonar/plugins/python/PythonRuleRepository.java [hunk 3]
```diff
@@ -19,14 +19,20 @@
...
+import java.io.InputStreamReader;
 import java.net.URL;
 import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Paths;
 import java.util.Locale;
 import java.util.Set;
+import java.util.stream.Collectors;
 import javax.annotation.Nullable;
+import org.apache.commons.io.IOUtils;
```
Comment (SonarTech) [inline sonar-python-plugin/src/main/java/org/sonar/plugins/python/PythonRuleRepository.java]: MINOR Remove this unused import 'org.apache.commons.io.IOUtils'. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AUselessImportCheck)

File: sonar-python-plugin/src/main/java/org/sonar/plugins/python/PythonRuleRepository.java [hunk 4]
```diff
@@ -19,14 +19,20 @@
  */
 package org.sonar.plugins.python;
 
-import com.google.common.io.Resources;
 import com.google.gson.Gson;
+import java.io.BufferedReader;
 import java.io.IOException;
+import java.io.InputStream;
```
Comment (SonarTech) [inline sonar-python-plugin/src/main/java/org/sonar/plugins/python/PythonRuleRepository.java]: MINOR Remove this unused import 'java.io.InputStream'. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AUselessImportCheck)

File: sonar-python-plugin/src/main/java/org/sonar/plugins/python/PythonRuleRepository.java [hunk 5]
```diff
@@ -19,14 +19,20 @@
...
+import java.io.InputStreamReader;
 import java.net.URL;
 import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
```
Comment (SonarTech) [inline sonar-python-plugin/src/main/java/org/sonar/plugins/python/PythonRuleRepository.java]: MINOR Remove this unused import 'java.nio.file.Files'. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AUselessImportCheck)

File: sonar-python-plugin/src/main/java/org/sonar/plugins/python/PythonRuleRepository.java [hunk 6]
```diff
@@ -19,14 +19,20 @@
...
+import java.nio.file.Paths;
```
Comment (SonarTech) [inline sonar-python-plugin/src/main/java/org/sonar/plugins/python/PythonRuleRepository.java]: MINOR Remove this unused import 'java.nio.file.Paths'. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AUselessImportCheck)

**Comment by SonarTech** [comment]
SonarQube analysis reported 31 issues
* ![MINOR][MINOR] 31 minor

Watch the comments in this conversation to review them.

#### Top 10 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MINOR][MINOR] [CollapsibleIfStatementsCheck.java#L63](https://github.com/SonarSource/sonar-python/blob/95bab8f2d4cdf0374165a0143be8760ea3a897ef/python-checks/src/main/java/org/sonar/python/checks/CollapsibleIfStatementsCheck.java#L63): Remove this use of "AstSelect"; it is deprecated. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)
1. ![MINOR][MINOR] [CollapsibleIfStatementsCheck.java#L63](https://github.com/SonarSource/sonar-python/blob/95bab8f2d4cdf0374165a0143be8760ea3a897ef/python-checks/src/main/java/org/sonar/python/checks/CollapsibleIfStatementsCheck.java#L63): Remove this use of "select"; it is deprecated. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)
1. ![MINOR][MINOR] [DuplicatedMethodFieldNamesCheck.java#L88](https://github.com/SonarSource/sonar-python/blob/95bab8f2d4cdf0374165a0143be8760ea3a897ef/python-checks/src/main/java/org/sonar/python/checks/DuplicatedMethodFieldNamesCheck.java#L88): Remove this use of "AstSelect"; it is deprecated. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)
1. ![MINOR][MINOR] [DuplicatedMethodFieldNamesCheck.java#L88](https://github.com/SonarSource/sonar-python/blob/95bab8f2d4cdf0374165a0143be8760ea3a897ef/python-checks/src/main/java/org/sonar/python/checks/DuplicatedMethodFieldNamesCheck.java#L88): Remove this use of "select"; it is deprecated. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)
1. ![MINOR][MINOR] [EmptyNestedBlockCheck.java#L51](https://github.com/SonarSource/sonar-python/blob/95bab8f2d4cdf0374165a0143be8760ea3a897ef/python-checks/src/main/java/org/sonar/python/checks/EmptyNestedBlockCheck.java#L51): Remove this use of "AstSelect"; it is deprecated. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)
1. ![MINOR][MINOR] [EmptyNestedBlockCheck.java#L51](https://github.com/SonarSource/sonar-python/blob/95bab8f2d4cdf0374165a0143be8760ea3a897ef/python-checks/src/main/java/org/sonar/python/checks/EmptyNestedBlockCheck.java#L51): Remove this use of "select"; it is deprecated. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)
1. ![MINOR][MINOR] [EmptyNestedBlockCheck.java#L52](https://github.com/SonarSource/sonar-python/blob/95bab8f2d4cdf0374165a0143be8760ea3a897ef/python-checks/src/main/java/org/sonar/python/checks/EmptyNestedBlockCheck.java#L52): Remove this use of "AstSelect"; it is deprecated. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)
1. ![MINOR][MINOR] [EmptyNestedBlockCheck.java#L54](https://github.com/SonarSource/sonar-python/blob/95bab8f2d4cdf0374165a0143be8760ea3a897ef/python-checks/src/main/java/org/sonar/python/checks/EmptyNestedBlockCheck.java#L54): Remove this use of "AstSelect"; it is deprecated. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)
1. ![MINOR][MINOR] [EmptyNestedBlockCheck.java#L61](https://github.com/SonarSource/sonar-python/blob/95bab8f2d4cdf0374165a0143be8760ea3a897ef/python-checks/src/main/java/org/sonar/python/checks/EmptyNestedBlockCheck.java#L61): Remove this use of "AstSelect"; it is deprecated. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)
1. ![MINOR][MINOR] [NeedlessPassCheck.java#L65](https://github.com/SonarSource/sonar-python/blob/95bab8f2d4cdf0374165a0143be8760ea3a897ef/python-checks/src/main/java/org/sonar/python/checks/NeedlessPassCheck.java#L65): Remove this use of "AstSelect"; it is deprecated. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)

[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Comment by SonarTech** [comment]
SonarQube analysis reported 27 issues
* ![MINOR][MINOR] 27 minor

Watch the comments in this conversation to review them.

#### Top 10 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MINOR][MINOR] [CollapsibleIfStatementsCheck.java#L63](https://github.com/SonarSource/sonar-python/blob/0d96072120bd5b7b89a349d4adc857729735535d/python-checks/src/main/java/org/sonar/python/checks/CollapsibleIfStatementsCheck.java#L63): Remove this use of "AstSelect"; it is deprecated. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)
1. ![MINOR][MINOR] [CollapsibleIfStatementsCheck.java#L63](https://github.com/SonarSource/sonar-python/blob/0d96072120bd5b7b89a349d4adc857729735535d/python-checks/src/main/java/org/sonar/python/checks/CollapsibleIfStatementsCheck.java#L63): Remove this use of "select"; it is deprecated. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)
1. ![MINOR][MINOR] [DuplicatedMethodFieldNamesCheck.java#L88](https://github.com/SonarSource/sonar-python/blob/0d96072120bd5b7b89a349d4adc857729735535d/python-checks/src/main/java/org/sonar/python/checks/DuplicatedMethodFieldNamesCheck.java#L88): Remove this use of "AstSelect"; it is deprecated. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)
1. ![MINOR][MINOR] [DuplicatedMethodFieldNamesCheck.java#L88](https://github.com/SonarSource/sonar-python/blob/0d96072120bd5b7b89a349d4adc857729735535d/python-checks/src/main/java/org/sonar/python/checks/DuplicatedMethodFieldNamesCheck.java#L88): Remove this use of "select"; it is deprecated. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)
1. ![MINOR][MINOR] [EmptyNestedBlockCheck.java#L51](https://github.com/SonarSource/sonar-python/blob/0d96072120bd5b7b89a349d4adc857729735535d/python-checks/src/main/java/org/sonar/python/checks/EmptyNestedBlockCheck.java#L51): Remove this use of "AstSelect"; it is deprecated. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)
1. ![MINOR][MINOR] [EmptyNestedBlockCheck.java#L51](https://github.com/SonarSource/sonar-python/blob/0d96072120bd5b7b89a349d4adc857729735535d/python-checks/src/main/java/org/sonar/python/checks/EmptyNestedBlockCheck.java#L51): Remove this use of "select"; it is deprecated. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)
1. ![MINOR][MINOR] [EmptyNestedBlockCheck.java#L52](https://github.com/SonarSource/sonar-python/blob/0d96072120bd5b7b89a349d4adc857729735535d/python-checks/src/main/java/org/sonar/python/checks/EmptyNestedBlockCheck.java#L52): Remove this use of "AstSelect"; it is deprecated. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)
1. ![MINOR][MINOR] [EmptyNestedBlockCheck.java#L54](https://github.com/SonarSource/sonar-python/blob/0d96072120bd5b7b89a349d4adc857729735535d/python-checks/src/main/java/org/sonar/python/checks/EmptyNestedBlockCheck.java#L54): Remove this use of "AstSelect"; it is deprecated. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)
1. ![MINOR][MINOR] [EmptyNestedBlockCheck.java#L61](https://github.com/SonarSource/sonar-python/blob/0d96072120bd5b7b89a349d4adc857729735535d/python-checks/src/main/java/org/sonar/python/checks/EmptyNestedBlockCheck.java#L61): Remove this use of "AstSelect"; it is deprecated. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)
1. ![MINOR][MINOR] [NeedlessPassCheck.java#L65](https://github.com/SonarSource/sonar-python/blob/0d96072120bd5b7b89a349d4adc857729735535d/python-checks/src/main/java/org/sonar/python/checks/NeedlessPassCheck.java#L65): Remove this use of "AstSelect"; it is deprecated. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)

[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

File: python-squid/src/main/java/org/sonar/python/DocstringExtractor.java [hunk 7]
```diff
@@ -36,14 +37,19 @@
  */
 public class DocstringExtractor {
 
-  public static final Set<AstNodeType> DOCUMENTABLE_NODE_TYPES = ImmutableSet.<AstNodeType>of(
-    PythonGrammar.FILE_INPUT,
-    PythonGrammar.FUNCDEF,
-    PythonGrammar.CLASSDEF);
+  public static final Set<AstNodeType> DOCUMENTABLE_NODE_TYPES = initializeDocumentableNodeTypes();
```
Comment (SonarTech) [inline python-squid/src/main/java/org/sonar/python/DocstringExtractor.java]: MINOR Make this member "protected". [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS2386)

**Comment by SonarTech** [comment]
SonarQube analysis reported 27 issues
* ![MINOR][MINOR] 27 minor

Watch the comments in this conversation to review them.

#### Top 10 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MINOR][MINOR] [CollapsibleIfStatementsCheck.java#L63](https://github.com/SonarSource/sonar-python/blob/d84430d39a2f0ffb809f70ad1993e00bd11f6d3d/python-checks/src/main/java/org/sonar/python/checks/CollapsibleIfStatementsCheck.java#L63): Remove this use of "AstSelect"; it is deprecated. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)
1. ![MINOR][MINOR] [CollapsibleIfStatementsCheck.java#L63](https://github.com/SonarSource/sonar-python/blob/d84430d39a2f0ffb809f70ad1993e00bd11f6d3d/python-checks/src/main/java/org/sonar/python/checks/CollapsibleIfStatementsCheck.java#L63): Remove this use of "select"; it is deprecated. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)
1. ![MINOR][MINOR] [DuplicatedMethodFieldNamesCheck.java#L88](https://github.com/SonarSource/sonar-python/blob/d84430d39a2f0ffb809f70ad1993e00bd11f6d3d/python-checks/src/main/java/org/sonar/python/checks/DuplicatedMethodFieldNamesCheck.java#L88): Remove this use of "AstSelect"; it is deprecated. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)
1. ![MINOR][MINOR] [DuplicatedMethodFieldNamesCheck.java#L88](https://github.com/SonarSource/sonar-python/blob/d84430d39a2f0ffb809f70ad1993e00bd11f6d3d/python-checks/src/main/java/org/sonar/python/checks/DuplicatedMethodFieldNamesCheck.java#L88): Remove this use of "select"; it is deprecated. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)
1. ![MINOR][MINOR] [EmptyNestedBlockCheck.java#L51](https://github.com/SonarSource/sonar-python/blob/d84430d39a2f0ffb809f70ad1993e00bd11f6d3d/python-checks/src/main/java/org/sonar/python/checks/EmptyNestedBlockCheck.java#L51): Remove this use of "AstSelect"; it is deprecated. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)
1. ![MINOR][MINOR] [EmptyNestedBlockCheck.java#L51](https://github.com/SonarSource/sonar-python/blob/d84430d39a2f0ffb809f70ad1993e00bd11f6d3d/python-checks/src/main/java/org/sonar/python/checks/EmptyNestedBlockCheck.java#L51): Remove this use of "select"; it is deprecated. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)
1. ![MINOR][MINOR] [EmptyNestedBlockCheck.java#L52](https://github.com/SonarSource/sonar-python/blob/d84430d39a2f0ffb809f70ad1993e00bd11f6d3d/python-checks/src/main/java/org/sonar/python/checks/EmptyNestedBlockCheck.java#L52): Remove this use of "AstSelect"; it is deprecated. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)
1. ![MINOR][MINOR] [EmptyNestedBlockCheck.java#L54](https://github.com/SonarSource/sonar-python/blob/d84430d39a2f0ffb809f70ad1993e00bd11f6d3d/python-checks/src/main/java/org/sonar/python/checks/EmptyNestedBlockCheck.java#L54): Remove this use of "AstSelect"; it is deprecated. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)
1. ![MINOR][MINOR] [EmptyNestedBlockCheck.java#L61](https://github.com/SonarSource/sonar-python/blob/d84430d39a2f0ffb809f70ad1993e00bd11f6d3d/python-checks/src/main/java/org/sonar/python/checks/EmptyNestedBlockCheck.java#L61): Remove this use of "AstSelect"; it is deprecated. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)
1. ![MINOR][MINOR] [NeedlessPassCheck.java#L65](https://github.com/SonarSource/sonar-python/blob/d84430d39a2f0ffb809f70ad1993e00bd11f6d3d/python-checks/src/main/java/org/sonar/python/checks/NeedlessPassCheck.java#L65): Remove this use of "AstSelect"; it is deprecated. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)

[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

File: sonar-python-plugin/src/main/java/org/sonar/plugins/python/PythonProfile.java (hunk 1 above)
Comment (SonarTech) [inline sonar-python-plugin/src/main/java/org/sonar/plugins/python/PythonProfile.java]: MINOR Refactor your code to get this URI from a customizable parameter. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1075)

**Comment by SonarTech** [comment]
SonarQube analysis reported 28 issues
* ![MINOR][MINOR] 28 minor

Watch the comments in this conversation to review them.

#### Top 10 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MINOR][MINOR] [CollapsibleIfStatementsCheck.java#L63](https://github.com/SonarSource/sonar-python/blob/d84430d39a2f0ffb809f70ad1993e00bd11f6d3d/python-checks/src/main/java/org/sonar/python/checks/CollapsibleIfStatementsCheck.java#L63): Remove this use of "AstSelect"; it is deprecated. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)
1. ![MINOR][MINOR] [CollapsibleIfStatementsCheck.java#L63](https://github.com/SonarSource/sonar-python/blob/d84430d39a2f0ffb809f70ad1993e00bd11f6d3d/python-checks/src/main/java/org/sonar/python/checks/CollapsibleIfStatementsCheck.java#L63): Remove this use of "select"; it is deprecated. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)
1. ![MINOR][MINOR] [DuplicatedMethodFieldNamesCheck.java#L88](https://github.com/SonarSource/sonar-python/blob/d84430d39a2f0ffb809f70ad1993e00bd11f6d3d/python-checks/src/main/java/org/sonar/python/checks/DuplicatedMethodFieldNamesCheck.java#L88): Remove this use of "AstSelect"; it is deprecated. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)
1. ![MINOR][MINOR] [DuplicatedMethodFieldNamesCheck.java#L88](https://github.com/SonarSource/sonar-python/blob/d84430d39a2f0ffb809f70ad1993e00bd11f6d3d/python-checks/src/main/java/org/sonar/python/checks/DuplicatedMethodFieldNamesCheck.java#L88): Remove this use of "select"; it is deprecated. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)
1. ![MINOR][MINOR] [EmptyNestedBlockCheck.java#L51](https://github.com/SonarSource/sonar-python/blob/d84430d39a2f0ffb809f70ad1993e00bd11f6d3d/python-checks/src/main/java/org/sonar/python/checks/EmptyNestedBlockCheck.java#L51): Remove this use of "AstSelect"; it is deprecated. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)
1. ![MINOR][MINOR] [EmptyNestedBlockCheck.java#L51](https://github.com/SonarSource/sonar-python/blob/d84430d39a2f0ffb809f70ad1993e00bd11f6d3d/python-checks/src/main/java/org/sonar/python/checks/EmptyNestedBlockCheck.java#L51): Remove this use of "select"; it is deprecated. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)
1. ![MINOR][MINOR] [EmptyNestedBlockCheck.java#L52](https://github.com/SonarSource/sonar-python/blob/d84430d39a2f0ffb809f70ad1993e00bd11f6d3d/python-checks/src/main/java/org/sonar/python/checks/EmptyNestedBlockCheck.java#L52): Remove this use of "AstSelect"; it is deprecated. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)
1. ![MINOR][MINOR] [EmptyNestedBlockCheck.java#L54](https://github.com/SonarSource/sonar-python/blob/d84430d39a2f0ffb809f70ad1993e00bd11f6d3d/python-checks/src/main/java/org/sonar/python/checks/EmptyNestedBlockCheck.java#L54): Remove this use of "AstSelect"; it is deprecated. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)
1. ![MINOR][MINOR] [EmptyNestedBlockCheck.java#L61](https://github.com/SonarSource/sonar-python/blob/d84430d39a2f0ffb809f70ad1993e00bd11f6d3d/python-checks/src/main/java/org/sonar/python/checks/EmptyNestedBlockCheck.java#L61): Remove this use of "AstSelect"; it is deprecated. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)
1. ![MINOR][MINOR] [NeedlessPassCheck.java#L65](https://github.com/SonarSource/sonar-python/blob/d84430d39a2f0ffb809f70ad1993e00bd11f6d3d/python-checks/src/main/java/org/sonar/python/checks/NeedlessPassCheck.java#L65): Remove this use of "AstSelect"; it is deprecated. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)

[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Comment by pynicolas** [comment]
The content of this PR was moved to other PRs:
* #118 
* #119 
* #120 
Thanks @dbmeneses !

**Pull request closed without merging**